server/index.ts

```diff
@@ -72,15 +76,15 @@ io.on('connection', socket => {
 		}
 	});
 
-	socket.on(SocketEvent.PLAY, async ({ cards }: { cards: Card[] }) => {
+	socket.on(SocketEvent.PLAY, async (data: { cards: Card[] }) => {
```

Comment:
I noticed this as well and would be fine with leaving it at {cards} in case we want to send further information along in the future.

Reply:
nvm, misread